daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/TypeCalculator.scala

```diff
@@ -163,7 +163,7 @@ class KeysetValueTypeCalculatorOrdered(valueMap: HashMap[DataValuePrimitive, Dat
   override def outputTypeCalc(x: DataValuePrimitive, xType: NodeInfo.Kind): (DataValuePrimitiveNullable, Maybe[Error]) = {
     unparseMap.get(x) match {
       case Some(v) => (v, Maybe.Nope)
-      case None => (DataValue.NoValue, One(s"Value ${x} not found in keyset-value mapping"))
+      case None => (DataValue.NoValue, One(s"Value ${x} not found in enumeration dfdlx:repValues"))
```

Comment:
Is there a reason why the message here at line 166 is not identical to the message at line 156?  That is, why does the other message mention only "enumeration" while this message mentions both "enumeration" and "dfdlx:repValues"?

BTW, I think the order should be reversed: "not found in dfdlx:repValues enumerations".

Reply:
You are right that they should be reversed.  Had the order mixed up in my head.  They aren't identical as on the inputTypeCalc side we are looking for a repValue and the outputTypeCalc side we are looking at the enumeration value.